app/templates/components/add-env-var.hbs

```diff
@@ -1,21 +1,48 @@
-<form {{action (perform save) on='submit'}}>
-  <div class="form-elem">
-    {{input value=name class="env-name" key-up=(action 'nameChanged') placeholder="Name"}}
-    {{#if nameIsBlank }}
-      <p class="form-error-message">Name cannot be blank</p>
-    {{/if}}
+{{#travis-form
+  onSubmit=(perform save)
+  as |form|
+}}
+  <div class='form-elem'>
+    {{#form.field
+      label='Name'
+      value=name
+      required=true
+      showValidationStatusIcons=false
+      as |field|
+    }}
+      {{field.input class='env-name' placeholder='Name' onChange=(action (mut name))}}
+    {{/form.field}}
   </div>
-  <div class="form-elem">
-    {{input value=value class="env-value" placeholder="Value"}}
+  <div class='form-elem'>
+   {{#form.field
+      label='Value'
+      value=value
+      required=true
+      showValidationStatusIcons=false
+      as |field|
+    }}
+      {{field.input  class='env-value' placeholder='Value' onChange=(action (mut value))}}
+    {{/form.field}}
   </div>
-  <div class="form-elem">
-    {{travis-switch active=public description="Display value in build log" onToggle=(action (mut public))}}
+  <div class='form-elem'>
+    {{#form.field
+      label='Display value in build log'
+      class='travis-form__field--without-frame travis-form__field--switch'
```

Comment:
1) It would be better to set the `travis-form__field--without-frame` class by setting `disableFrame=true` property (they are bound inside component), to keep it encapsulated.
2) The `travis-form__field--switch` class should be set as a root class in component definition, because all instances of this component must have this class.